my-app/src/main/java/com/mycompany/app/findInverseSquareRoot.java

```diff
@@ -0,0 +1,26 @@
+package com.mycompany.app;
+
+public class findInverseSquareRoot {
+
+		public static float Q_rsqurt(float x) {
+			if (x <= 0) {
+	        	throw new ArithmeticException("Input should be bigger than 0.");
+	        }
+	        int i;
+	        float x2, y;
+	        float thereehalfs = 1.5f;
+	
+	        x2 = x * 0.5f;
+	        y = x;
+	        i = Float.floatToIntBits(y);
+	        i = 0x5f3759df - (i >> 1);
```

Comment:
What's the purpose of this number? Put in a comment.

Reply:
Done